guides/source/caching_with_rails.md

```diff
@@ -179,6 +179,24 @@ With `touch` set to true, any action which changes `updated_at` for a game
 record will also change it for the associated product, thereby expiring the
 cache.
 
+### Shared Partial Caching
+
+It is possible to share partials and associated caching between files with different mime types. For example shared partial caching allows template writers to share a partial between html and javascript files. Because of the way templates are collected in the template resolver file paths that only include the template language extension, not the mime type, can be used across mime types. So a request for an html file or js file that includes the following code:
+
+```ruby
+render(partial: 'hotels/hotel', collection: @hotels, cached: true)
+```
+
+Will load a file named `hotels/hotel.erb`.
+
+Another option is to include the full filename of the partial to include.
+
+```ruby
+render(partial: 'hotels/hotel.html.erb', collection: @hotels, cached: true)
+```
+
+Will load a file named `hotels/hotel.html.erb` in any file mime type, for example you could include this partial in a javascript file..
```

Comment:
You have an extra period here and JavaScript should be written as it's proper name.

> Will load a file named `hotels/hotel.html.erb` in any file mime type. You could include this partial in a JavaScript file.